frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -193,6 +206,30 @@ export const PaymentInput = (): JSX.Element => {
 
         {watchedEnabled && (
           <>
+            <FormControl
+              isReadOnly={paymentsMutation.isLoading}
+              isInvalid={!!errors.description}
+              isRequired
+            >
+              <Flex>
+                <FormLabel>Description</FormLabel>
```

Comment:
As above, follow @staceytan1998's terminology for better user experience:
```suggestion
                <FormLabel>Name</FormLabel>
```

Reply:
[aa9d346](https://github.com/opengovsg/FormSG/pull/5998/commits/aa9d3465942aec055d626177146e41c6b4aed2d2)